modules/main-rules/pages/broken-detachments.adoc

```diff
@@ -18,10 +17,12 @@ This is the only type of move you may make with a Broken detachment.
 [[retreat-move]]
 == Retreat move
 
-The Retreat move is a single move in which you may move all of the Broken units up to 20 cm -- regardless of the actual Speed values of units.
+The Retreat move is a single move in which you may move all mobile units in the Broken detachment up to 20 cm -- regardless of their actual Speed values.
 
-* You may make the Retreat move in any direction, subject to the normal rules for terrain effects on movement.
-* If you end a Retreat move such that any Broken units are within 15 cm of *any* enemy unit footnote:[Even Broken enemy units.], then they are *destroyed* -- remove those units as casualties.
+* You may use the Retreat move to move the units in any direction, subject to the normal rules for terrain effects on movement.
+* You may 'move' units 0 cm if you wish.
+Or, it may be that some units are not mobile, so you can't move them anyway.footnote:[Non-mobile units include those which have a Speed value of '0 cm' or `-`, or that are subject to a rule that says that you 'may not move' them. The Retreat move rules do not override this.]
+* If you end a Retreat move such that any Broken units are within 15 cm of *any* enemy unitfootnote:[Even Broken enemy units.], then they are *destroyed* -- remove those units as casualties.
```

Comment:
Continuing the conversation from `heavy-barrage.adoc`, about 'may not move'.

I've mentioned "Speed value of '0 cm' or `-`" here, but I think that if this is ever a thing, it should always be `-` (no value) and never '0 cm' (value of zero). That would work better with the text "regardless of their actual Speed values".

Do you agree? If so, we can remove the `0 cm` part of this edit.

Reply:
I strongly suggest leaving out all mention of Speed 0 cm or '-' units for now. If we ever add them, we will need to redo a whole bunch of different areas of the rules (Data sheets, the Movement phase, Retreat moves, Overwatch orders, roads, Assault orders and assault moves, Distortion Cannon, potentially Transportation etc.), and almost all the standard scenarios as they require units to move onto the table. 

At some point I had a look at this area for adding things like Tyranid Hive nests, and it means changing some pretty fundamental parts of the rules and scenarios!

Given this, would it be simpler to keep the retreat text relatively close to the previous version, and add a note reminding players that rules that prevent or stop movement ('for example Dangerous terrain tests, reloading Heavy Barrage units yadda yadda') also affect Retreat movement?

Reply:
>I strongly suggest leaving out all mention of Speed 0 cm or '-' units for now.

Ah, okay — I hadn't thought it through and only had in mind the loose idea of 'immobile' units that are deployed from transports, or that are set up on the board during deployment. I was trying to think ahead, but I'm happy to leave this out — especially given the issues you mention!

I'll tweak it again.

Reply:
Conversations outdated due to new tweaks, so I'll resolve.

Reply:
> only had in mind the loose idea of 'immobile' units that are deployed from transports, or that are set up on the board during deployment

No worries, I started at the same point, and then decided to leave it for a future date as it involved more changes the more I looked at it!